themes/Hacks2013/style.css

```diff
@@ -24,7 +24,7 @@ Medium Gray     #e5e5e5
 /* =Reset defaults */
 header, hgroup, nav, section, article, aside, footer, main { display: block; }
 html, body, form, fieldset, legend, h1, h2, h3, h4, h5, h6, dd { margin: 0; padding: 0; }
-p, pre, blockquote, ul, ol, dl, table { margin: 0 0 1.4em; padding: 0; }
+p, pre, blockquote, ul, ol, dl, table { margin: 0 0 1.4em; padding: 0; line-height: 1.6em;}
```

Comment:
Use <a href="http://meyerweb.com/eric/thoughts/2006/02/08/unitless-line-heights/">unitless line-height</code>, though a 1.6 line-height is declared for the body further down (and line-height inherits) so I'm not sure how effective it is to declare it in the reset.